### DISCUSSION TITLE
Transcriptions from Audio/Video that is searchable - via Whisper

### DISCUSSION THREAD

**Discussion by eleazar7** [spacedriveapp/spacedrive#1159] [Feature Request]
Can you search for Transcriptions from the Audio/Video files? -  

You could use Whisper to generate the transcriptions! 

Also, facial recognition and object detection in photos?
Also timestamp-based clips from videos/audio in which you stated, that there would be an encoder for media files so this fits perfectly 

Please add this!

**Comment by jamiepine** [comment]
All of this is planned, will look into Whisper🙏🏻
